Bugzilla/Install.pm

```diff
@@ -175,6 +175,12 @@ sub SETTINGS {
         default => 'off',
         category => 'API'
     },
+    {
+        name     => 'use_elasticsearch',
```

Comment:
nope! those are two different things, one is user setting and the other is a admin parameter. They are two very different systems. :-/